README.md

```diff
@@ -80,10 +85,9 @@ We gratefully acknowledge support from the following institutions:
 
 
 [OpenCoarrays]: http://www.opencoarrays.org
-[ABI]: https://gcc.gnu.org/onlinedocs/gfortran/Function-ABI-Documentation.html#Function-ABI-Documentation
+[ABI]: https://gcc.gnu.org/onlinedocs/gfortran/Coarray-Programming.html#Coarray-Programming
 [TS 18508]: http://isotc.iso.org/livelink/livelink?func=ll&objId=16769292&objAction=Open
 [MPI]: http://www.mpi-forum.org
-[ABI]: https://gcc.gnu.org/onlinedocs/gfortran/Function-ABI-Documentation.html#Function-ABI-Documentation
 [GCC]: http://gcc.gnu.org
```

Comment:
This is the change @rouson requested on gitter.im It turns out that the ABI link was defined twice.